### PR TITLE
Added a few more Scholia mappings

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

File: src/bioregistry/data/bioregistry.json [hunk 1]
```diff
@@ -40707,4 +40713,4 @@
       "prefix": "ZP"
     }
   }
-}
\ No newline at end of file
+}
```
Comment (cthoyt) [inline src/bioregistry/data/bioregistry.json:40716]: Normally I would agree but this file is managed by a python script so it gets dumped each time with no newline at the end

Comment (cthoyt) [reply src/bioregistry/data/bioregistry.json]: Fixed in 324dc2c

Comment (egonw) [reply src/bioregistry/data/bioregistry.json]: I don't even know how to solve this with my editor (`kate`).

Comment (egonw) [reply src/bioregistry/data/bioregistry.json]: (but I'll make care to use `git add -p` next time)

**Comment by cthoyt** [comment]
@egonw idk why I didn't go with your suggestion before, but it's finally fixed in #2067

**Pull request merged**